### PR TITLE
Integration - YouTube regrets 2021

### PR DESCRIPTION
**Pull request by chris-lawton**
Closes #6949, #6953, #6954, #6955, #6956, #6963, #6964, #6967

Integration branch for the YouTube regrets 2021 landing page

### CODE REVIEW

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/hero.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{% load static %}
+<div class="container dark-theme mb-3">
+  <div class="hero">
+    <div class="row">
+      <div class="col-12">
+        <img class="hero__image hero__image--mobile" src="{% static "_images/youtube-regrets/regrets-reporter/report-findings-hero-mobile.png" %}" alt="">
+        <img class="hero__image hero__image--desktop" src="{% static "_images/youtube-regrets/regrets-reporter/report-findings-hero-desktop.png" %}" alt="">
+        <div class="hero__guts">
+          <h1 class="mb-3 hero-heading">YouTube Regrets</h1>
+          <p class="body-large">Mozilla and 37,00 YouTube users conducted a study to better understand harmful YouTube recommendations. <span class="fst-italic">This is what we learned</span>.</p>
```
Comment (KalobTaulien) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/hero.html:10]: Needs translation template tag

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/intro.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+      <p class="mb-5 pb-4">Yet when confronted about its recommendation algorithm, <span class="fw-bold">YouTube’s routine response is to deny and deflect</span>.</p>
+      <p>Faced with this continuous refusal, Mozilla built a browser extension, <a class="fw-bold" href="#">RegretsReporter</a>, that allows people to donate data about the YouTube videos they regret watching on YouTube. Through one of the biggest crowdsourced investigations into YouTube, we’ve uncovered new information that warrants urgent attention and action from lawmakers and the public.</p>
+      <p class="fw-bold">We now know that YouTube is recommending videos that violate their very own policies and harm users around the world — and that needs to stop.</p>
```
Comment (KalobTaulien) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/intro.html:7]: Also needs `{% translate ".." %}` here

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/our-research.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      </div>
+      <div class="col-md-4 col-xs-12 stat">
+        <p class="stat__number" id="countries-count-up">91</p>
+        <p class="stat__label">{% trans "Countries" %}</p>
+      </div>
+    </div>
+
+    <div class="row justify-content-center mb-5">
+      <div class="col-12 col-md-8 text-center">
+        {% blocktrans %}
```
Comment (KalobTaulien) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/our-research.html:36]: With `{% blocktrans %}` let's trim it with `{% blocktrans trimmed %}`

File: source/images/youtube-regrets/close.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg width="69" height="69" fill="none" xmlns="http://www.w3.org/2000/svg"><path stroke="#000" d="M43.9531 43.6524L26.2025 25.9018M43.6563 25.9034L25.9057 43.6539"/><circle cx="34.9314" cy="34.9258" r="23.5" transform="rotate(-45 34.9314 34.9258)" stroke="#000"/></svg>
```
Comment (KalobTaulien) [inline source/images/youtube-regrets/close.svg:1]: SVG's might be compressable using `svgo`.

**Review by KalobTaulien — APPROVED** [review]
One major thing I noticed was the drawers don't work at all on localhost. Not sure if that's intentional at the moment or not, but I see we have a bunch of JavaScript implemented to make it work. And there are sections of content that haven't been properly filled in from the design yet (but that could be because we didn't have the content at the time this PR was submitted, I'll admit I'm a bit behind on that one). 

I'm going to approve this because we need to get this onto a review app for testing ASAP, even though it still needs more work. This may result in a bunch of follow-up tickets.

File: network-api/networkapi/views.py [hunk 5]
```diff
@@ -19,3 +19,6 @@ def review_app_help_view(request):
         return render(request, 'reviewapp-help.html')
     else:
         return HttpResponse(status=404)
+
+def YoutubeRegrets2021View(request):
+   return render(request, 'wagtailpages/pages/youtube-regrets-2021/youtube_regrets_2021.html')
```
Comment (KalobTaulien) [inline network-api/networkapi/views.py:24]:
```suggestion
    return render(request, 'wagtailpages/pages/youtube-regrets-2021/youtube_regrets_2021.html')
```

Flake8 needs a multiple of 4.

File: network-api/networkapi/views.py [hunk 6]
```diff
@@ -19,3 +19,6 @@ def review_app_help_view(request):
         return render(request, 'reviewapp-help.html')
     else:
         return HttpResponse(status=404)
+
```
Comment (KalobTaulien) [inline network-api/networkapi/views.py:22]:
```suggestion


```
Flake8 needs 2 blank lines.

**Review by KalobTaulien — CHANGES_REQUESTED** [review]
These changes need to happen before we can merge this in otherwise it doesn't pass our CI. You should be able to just click "Commit suggestion" in the GitHub UI as a fast way of applying the fixes.

**Review by KalobTaulien — APPROVED** [review]

File: source/sass/views/youtube-regrets-2021.scss [hunk 7]
```diff
@@ -0,0 +1,465 @@
...
+        position: absolute;
+        left: 0;
+        width: 18px;
+        height: 18px;
+      }
+
+      &--light {
+        &::before {
+          background-color: #FBD5D7;
+        }
```
Comment (KalobTaulien) [inline source/sass/views/youtube-regrets-2021.scss:309]:
```suggestion
        &::before {
          /* stylelint-disable */
          background-color: #FBD5D7;
          /* stylelint-enable */
        }
```

File: source/sass/views/youtube-regrets-2021.scss [hunk 8]
```diff
@@ -0,0 +1,465 @@
...
+      }
+
+      &--dark {
+        margin-right: 40px;
+
+        &::before {
+          background-color: #FC4147;
+        }
```
Comment (KalobTaulien) [inline source/sass/views/youtube-regrets-2021.scss:317]:
```suggestion
        &::before {
          /* stylelint-disable */
          background-color: #FC4147;
          /* stylelint-enable */
        }
```

File: source/sass/views/youtube-regrets-2021-carousel.scss [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+  }
+
+  &__item-text {
+    @include set-text-size(28px, 30px);
+    font-family: $font-zilla;
+    font-weight: 500;
+
+    span {
+      color: #d63126;
+    }
```
Comment (KalobTaulien) [inline source/sass/views/youtube-regrets-2021-carousel.scss:53]:
```suggestion
    span {
      /* stylelint-disable */
      color: #d63126;
      /* stylelint-enable */
    }
```

**Review by KalobTaulien — COMMENTED** [review]
One-off hex colors that don't need a variable [yet]. Disable style linting for these.

File: source/images/youtube-regrets/close.svg (hunk 4 above)
Comment (KalobTaulien) [reply source/images/youtube-regrets/close.svg:1]: Ticket here #6983

**Review by danielfmiranda — COMMENTED** [review]
Hi All! Thank you for tagging me in this PR, I have given it a look on my local machine and can confirm that the youtube-regrets-2021 path takes me to the landing page, and that the content on the landing page works as designed.

Awesome work, I am approving!

**Pull request merged**